pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -11,7 +11,7 @@
     "actiontec" = ps: with ps; [ ];
     "adguard" = ps: with ps; [ adguardhome ];
     "ads" = ps: with ps; [ pyads ];
-    "advantage_air" = ps: with ps; [ ]; # missing inputs: advantage_air
+    "advantage_air" = ps: with ps; [ advantage_air ];
```

Comment:
```suggestion
    "advantage_air" = ps: with ps; [ advantage-air ];
```

Reply:
This file is autogenerated, so please execute `parse-requirements.py` to update it.

Reply:
Not sure why this was marked resolved, it was still in need of the change.

Resolved in https://github.com/NixOS/nixpkgs/pull/108982.